src/test/java/org/fogbowcloud/manager/OrderStateTransitionerTest.java

```diff
@@ -84,73 +93,95 @@ public void testValidTransition() throws UnexpectedException {
         Order order = createOrder(originState);
         openOrdersList.addItem(order);
 
+        // exercise
         OrderStateTransitioner.transition(order, destinationState);
-        assertNull(openOrdersList.getNext());
-        assertEquals(order, spawningOrdersList.getNext());
+
+        // verify
+        Assert.assertNull(openOrdersList.getNext());
```

Comment:
Don't we need to verify the calls to the databaseManager mock?

Reply:
Not because currently this database script has been replaced by the method call 'mockReadOrdersFromDataBase()' of the super class 'BaseUnitTests'.